coretex/cli/commands/node.py

```diff
@@ -94,21 +87,75 @@ def update() -> None:
         ):
             return
 
-    click.echo("Stopping Coretex Node...")
     node_module.stop()
-    click.echo("Successfully stopped Coretex Node.")
 
-    click.echo("Starting Coretex Node...")
     node_module.start(f"{repository}:{tag}", config)
-    click.echo("Successfully started Coretex Node.")
+
+
+@click.command()
+@click.option("--verbose", is_flag = True, help = "Configure node settings manually.")
+def config(verbose: bool) -> None:
+    if node_module.isRunning():
+        if click.prompt("Node is already running. Do you wish to stop the Node? (Y/n)",
+            type = bool,
+            default = True,
+            show_default = False):
+            node_module.stop()
+
+        click.echo("If you wish to reconfigure your node, use coretex node stop commands first.")
+        return
+
+    config = loadConfig()
+
+    if isNodeConfigured(config):
+        if not click.prompt(
+            "Node configuration already exists. Would you like to update? (Y/n)",
+            type = bool,
+            default = True,
+            show_default = False
+        ):
+            return
+
+    click.echo("[Node Configuration]")
+
+    config["storagepath"] = Path.home() / ".coretex"
```

Comment:
Why is this not a part of the same logic flow as for the memory related configurations?